chalice/app.py

```diff
@@ -137,6 +137,18 @@ def dispatch(*args, **kwargs):
         method = app.current_request.method
         query_params = app.current_request.query_params
         req_body = app.current_request.raw_body if app.current_request._body is not None else None
+        source_ip = app.current_request.context['identity']['sourceIp']
+        content_length = app.current_request.headers.get('content-length')
+        user_agent = app.current_request.headers.get('user-agent')
+        app.log.info(
+            """[request] "%s %s", \{"source_ip":"%s", "content_length":"%s", "user-agent":"%s"\}, %s""",
```

Comment:
Can you explain this message format?

Reply:
I'm curious too. Is this documented somewhere?

Reply:
This would be a concatenation of [common log format](https://en.wikipedia.org/wiki/Common_Log_Format) with a JSON object. This is not a standard format. I would recommend either 1) going full JSON or 2) following something standard like [combined log format](https://httpd.apache.org/docs/1.3/logs.html#combined).